app/assets/stylesheets/simple-sidebar.scss

```diff
@@ -3,90 +3,122 @@
  * Copyright 2013-2016 Start Bootstrap
  * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap/blob/gh-pages/LICENSE)
  */
-/* Toggle Styles */
-/* Sidebar Styles */
-
-$sidebar-size: 200px;
 
 .menu-toggle {
-  &-opened {
-    margin: 0;
-    height: 36px;
-    width: 36px;
-    position: relative;
-    top: 12px;
-    right: 15px;
-    text-indent: 0;
-  }
-
-  &-closed {
-    margin-right: 15px;
-    height: 38px;
-    padding: 6px 8px;
-  }
+  margin-right: 0.5em;
+  text-indent: 0;
 }
 
-#wrapper {
-	padding-left: 0;
-}
 #sidebar-wrapper {
-	z-index: 2000;
-	position: fixed;
-	width: $sidebar-size;
-	height: 100%;
-	background: #000;
+  z-index: 2000;
+  position: fixed;
+  width: $sidebar-size;
+  height: 100%;
+  background: $sidebar-bg-default;
   overflow: auto;
-}
-#page-content-wrapper {
-	width: 100%;
-	position: absolute;
-  padding-top: 20px;
-
-  #main .row {
-    margin-left: 10px;
-    margin-right: 10px;
+
+  &::-webkit-scrollbar {
+    display: none;
   }
 }
+
 .sidebar-nav {
-	position: absolute;
-	top: 0;
-	width: $sidebar-size;
-	margin: 0;
-	padding: 0;
-	list-style: none;
-	li {
-		text-indent: 20px;
-		line-height: 40px;
-		a {
-			display: block;
-			text-decoration: none;
-			color: #999999;
-			&:hover {
-				text-decoration: none;
-				color: #444;
-				background: rgba(255,255,255,0.2);
-			}
-			&:active {
-				text-decoration: none;
-			}
-			&:focus {
-				text-decoration: none;
-			}
-		}
-	}
-	>.sidebar-brand {
-		height: 65px;
-		font-size: 18px;
-		line-height: 60px;
-		a {
-			color: #999999;
-			&:hover {
-				color: #fff;
-				background: none;
-			}
-		}
-	}
+  position: absolute;
+  top: 0;
+  width: $sidebar-size;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  font-size: 1em;
+
+  li {
+    text-indent: 1.2em;
+    line-height: 2.8em;
```

Comment:
It's a best practice not to use units on line-height. It can lead to unexpected behaviour on child elements https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0